frontend/src/v5/ui/routes/dashboard/projects/drawings/drawingsList/drawingsListItem/drawingsListItem.component.tsx

```diff
@@ -52,12 +53,13 @@ export const DrawingsListItem = memo(({
 	const { teamspace, project } = useParams<DashboardParams>();
 	const isMainList = useContext(IsMainList);
 	const isProjectAdmin = ProjectsHooksSelectors.selectIsProjectAdmin();
+	const drawingId = drawing._id;
 
 	const onChangeFavourite = ({ currentTarget: { checked } }) => {
 		if (checked) {
-			DrawingsActionsDispatchers.addFavourite(teamspace, project, drawing._id);
+			DrawingsActionsDispatchers.addFavourite(teamspace, project, drawingId);
```

Comment:
The drawing list doesn't look very good. I guess something got messed up during rebase 🫤

<img width="440" alt="image" src="https://github.com/3drepo/3drepo.io/assets/42032433/1590ef48-0f84-4684-a01e-d1891c449f77">